econml/iv/dml/_dml.py

```diff
@@ -526,7 +526,7 @@ def score(self, Y, T, Z, X=None, W=None, sample_weight=None):
             The MSE of the final CATE model on the new data.
```

Comment:
minor since groups aren't really used for scoring but they are not included in the docstring as parameters

Reply:
As mentioned in a previous comment, removed groups from scoring on the estimator since they do nothing